packages/decoder/tsconfig.json

```diff
@@ -10,9 +10,10 @@
     "baseUrl": ".",
     "lib": ["es2017"],
     "paths": {
-      "web3": ["../../node_modules/@types/web3/index", "node_modules/web3/index"],
-      "web3/types": ["../../node_modules/@types/web3/types", "node_modules/web3/types"],
-      "web3/eth/types": ["../../node_modules/@types/web3/eth/types", "node_modules/web3/eth/types"]
+      "web3-core": [
+        "../../node_modules/web3/node_modules/web3-core/types",
```

Comment:
Why do both of these paths need to be specified and not just one?

Reply:
Just a fallback?

Reply:
Yup. The first path is in monorepo context, second is when working w/ the `decoder` pkg independently.